backup/setup/schools.json

```diff
@@ -90,6 +90,9 @@
 			},
 			{
 				"$oid": "0000d186816abba584714c93"
+			},
```

Comment:
We should remove OIDC system from Paul-Gerhardt-Gymnasium.
That's what we have OIDC-Mock-School for.

Reply:
Done